### PR TITLE
🦋 Update ActiveDocs table with PF4

### PR DESCRIPTION
**Pull request by josemigallas**
[THREESCALE-9784: PF4 table toolbar](https://issues.redhat.com/browse/THREESCALE-9784)

Before:
Screenshot 2024-06-11 at 12 34 18

After:
https://github.com/3scale/porta/assets/11672286/8d698820-2ad3-40fd-867d-e11802d3b6d5

### CODE REVIEW

**Comment by mayorova** [comment]
I'm still reviewing but I noticed that clicking on the "API" set an empty value for the "sort" query is empty...

**Comment by josemigallas** [comment]
> I'm still reviewing but I noticed that clicking on the "API" set an empty value for the "sort" query is empty...

Fixed, it should not be sortable in the first place.

File: test/integration/admin/api_docs/account_api_docs_controller_test.rb [hunk 1]
```diff
@@ -102,10 +91,10 @@ def test_system_name_is_not_updated
 
       logout! && login!(provider, user: member)
 
-      get admin_api_docs_services_path
-      api_docs_service_ids = assigns(:api_docs_services).map(&:id)
-      assert_includes api_docs_service_ids, api_docs_service.id
-      assert_not_includes api_docs_service_ids, forbidden_api_docs_service.id
+      # get admin_api_docs_services_path
```
Comment (lvillen) [inline test/integration/admin/api_docs/account_api_docs_controller_test.rb:94]: Just a question, are these commented lines supposed to go?

**Review by lvillen — DISMISSED** [review]
Great work! 🏅

File: test/integration/admin/api_docs/account_api_docs_controller_test.rb (hunk 1 above)
Comment (josemigallas) [reply test/integration/admin/api_docs/account_api_docs_controller_test.rb]: Yep 👍🏻

**Review by lvillen — APPROVED** [review]

**Pull request merged**